ckan/views/user.py

```diff
@@ -295,6 +296,7 @@ def api_token_revoke(id: str, jti: str) -> Response:
 
 class EditView(MethodView):
     def _prepare(self, id: Optional[str]) -> tuple[Context, str]:
+        request.environ[u'__is_sensitive__'] = True
```

Comment:
passing values to middleware with request environment settings feels fragile and unrelated to response headers that will be added. Is this the best interface?

We'll also have to think about backwards compatibility for existing extensions, what will extension authors need to know about upgrading to support the next version of ckan with these changes?

Reply:
for now, i'll leave it out and come back to this when tests are green again.